woocommerce/woocommerce
# Poc/WooWizard assistant

### Changes proposed in this Pull Request:

Add a Woo Wizard AI Assistant which is accessible via typing "Woo Wizard" into the Command Palette.



### How to test the changes in this Pull Request:

1. Run `pnpm run build` within `plugins/woo-ai`.
2. Open your Jetpack-connected and Woo AI-enabled site and go to a blocks-based editor, ( Posts -> Add New Post ).
3. Type Ctrl+K to open up the Command Palette and start typing "wizard", you should see "Woo Wizard" pop up.
4. Start chatting with the bot! Type a message and hit Send, then wait.
5. There is already logic and UX for using a microphone input for speech-to-text, however, the backend implementation is not working yet, so don't bother trying to use it. :)

### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
